docs/api.md

```diff
@@ -951,6 +1151,15 @@ Market Response Object:
 | fundingRate          | The funding rate for the market.                                      |
 | globalIndexValue     | The global index value for the market.                                |
 | globalIndexTimestamp | The Unix timestamp (seconds) for the last update to the global index. |
+| minCollateral        | The minimum collaterization before getting liquidated eg: 1.07 |
+| decimals             | Corresponds to the position units eg: if decimals = 8, then 100000000 = 1 BTC |
+| minimumTickSize      | The minimum price amount eg: 0.01 (equal to $1)|
+| minimumOrderSize     | The minimum size, in position units, required for an order |
+| smallOrderThreshold  | The threshold, in position units, at which we charge different fees for takers |
+| makerFee             | The fee charged for the maker of an order eg -0.00025 (equal to -0.025%) |
```

Comment:
The `percentage fee`, differentiate from a flat one

Reply:
added